lib/components/Button/Button.module.scss

```diff
@@ -36,97 +35,78 @@ $outline-focus-border-width: 1px;
         font-size: var(--icon-size-xsmall);
     }
 
-    &:not(:disabled) {

```

Comment:
don't we want this for all the buttons?

Reply:
we still have this, I just removed it from here because it was unnecessary duplication. It is inlcuded below